app/jobs/notifications/remove_all_job.rb

```diff
@@ -2,8 +2,10 @@ module Notifications
   class RemoveAllJob < ApplicationJob
     queue_as :remove_all_notifications
 
-    def perform(notifiable_id, notifiable_type, action, service = Notifications::RemoveAll)
-      service.call(notifiable_id, notifiable_type, action)
+    def perform(notifiable_ids, notifiable_type, service = Notifications::RemoveAll)
+      return unless %w[Article Comment Mention].include?(notifiable_type)
+
+      service.call(Array.wrap(notifiable_ids), notifiable_type)
```

Comment:
Btw `Array()` (`Kernel#Array`) can be used in many of such cases too. `Array#wrap` and `Array()` have [some differences](https://api.rubyonrails.org/classes/Array.html#method-c-wrap), but they are similar.